### ISSUE TITLE
panic: interface conversion in jira.resourceIssueRead

### ISSUE THREAD

**Issue by fransf-wtax** [fourplusone/terraform-provider-jira#50]
Terraform version:
```
Terraform v1.1.5
on darwin_amd64
+ provider registry.terraform.io/fourplusone/jira v0.1.15
+ provider registry.terraform.io/hashicorp/aws v3.66.0
```

Getting this error:
```
╷
│ Error: Request cancelled
│ 
│   with module.clients-uat_wtax_co_cloudfront.jira_issue.dns_change_request,
│   on ../../modules/cloudfront_distribution/main.tf line 24, in resource "jira_issue" "dns_change_request":
│   24: resource "jira_issue" "dns_change_request" {
│ 
│ The plugin.(*GRPCProvider).ReadResource request was cancelled.
╵

Stack trace from the terraform-provider-jira_v0.1.15 plugin:

panic: interface conversion: interface {} is map[string]interface {}, not map[string]string

goroutine 68 [running]:
github.com/fourplusone/terraform-provider-jira/jira.resourceIssueRead(0xc000198690, 0x1a719e0, 0xc00009c168, 0xc000198690, 0x0)
        github.com/fourplusone/terraform-provider-jira/jira/resource_issue.go:200 +0xda4
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc0000ba280, 0xc00075a960, 0x1a719e0, 0xc00009c168, 0xc00009c3c8, 0x0, 0x0)
        github.com/hashicorp/terraform-plugin-sdk@v1.6.0/helper/schema/resource.go:455 +0x119
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ReadResource(0xc00009c068, 0x1e2d660, 0xc000157710, 0xc00075a780, 0xc00009c068, 0xc000157710, 0xc00010aa48)
        github.com/hashicorp/terraform-plugin-sdk@v1.6.0/internal/helper/plugin/grpc_provider.go:525 +0x3d8
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ReadResource_Handler(0x1b97ec0, 0xc00009c068, 0x1e2d660, 0xc000157710, 0xc0005f83c0, 0x0, 0x1e2d660, 0xc000157710, 0xc0007dedc0, 0x2c0)
        github.com/hashicorp/terraform-plugin-sdk@v1.6.0/internal/tfplugin5/tfplugin5.pb.go:3153 +0x217
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00013a000, 0x1e374e0, 0xc000603380, 0xc000250200, 0xc0001e8690, 0x251dc90, 0x0, 0x0, 0x0)
        google.golang.org/grpc@v1.23.0/server.go:995 +0x460
google.golang.org/grpc.(*Server).handleStream(0xc00013a000, 0x1e374e0, 0xc000603380, 0xc000250200, 0x0)
        google.golang.org/grpc@v1.23.0/server.go:1275 +0xd3d
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc000036120, 0xc00013a000, 0x1e374e0, 0xc000603380, 0xc000250200)
        google.golang.org/grpc@v1.23.0/server.go:710 +0xa1
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.23.0/server.go:708 +0xa1

Error: The terraform-provider-jira_v0.1.15 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**Comment by fransf-wtax** [comment]
@fourplusone Change in PR above works for me, but I am not at all familiar with Golang or the source code of this project, so let me know if more changes are needed (e.g. testcases). Thanks!

**Comment by rubenk** [comment]
I'm hitting this as well in v0.1.16 with the following code:
```
terraform {
  required_providers {
    jira = {
      source = "fourplusone/jira"
      version = "0.1.16"
    }
  }
}

provider jira {
}

data "jira_field" "epic_name" {
  name = "Epic Name"
}

resource "jira_issue" "example_epic" {
  issue_type = "Epic"
  project_key = "TMP"
  summary = "Epic Epic"
  fields = {
    (data.jira_field.epic_name.id) = "Example epic name"
  }
}

**Comment by fourplusone** [comment]
Fixed, since #51 is merged
